examples/lock-app/efr32/src/AppTask.cpp

```diff
@@ -254,9 +254,43 @@ CHIP_ERROR AppTask::Init()
                      endpointId);
         numberOfSupportedUsers = 10;
     }
+
+    uint8_t numberOfWeekdaySchedulesPerUserSupported = 0;
+    if (!DoorLockServer::Instance().GetNumberOfWeekDaySchedulesPerUserSupported(endpointId,
+                                                                                numberOfWeekdaySchedulesPerUserSupported))
+    {
+        ChipLogError(
+            Zcl,
+            "Unable to get number of supported weekday schedules when initializing lock endpoint, defaulting to 10 [endpointId=%d]",
+            endpointId);
+        numberOfWeekdaySchedulesPerUserSupported = 10;
```

Comment:
Please use named constants for these values. Shouldn't this be DOOR_LOCK_MAX_SCHEDULES?

Reply:
So DOOR_LOCK_MAX_SCHEDULES is a preprocessor directive that is used to initialize the schedule arrays. 
There is are also dynamic variables, that are taken from attributes from zap i.e. (mNumberOfWeekdaySchedulesPerUserSupported, mNumberOfYeardaySchedulesPerUserSupported)

There is a check in LockManager::Init, that makes sure that the dynamic variables do not exceed the array sizes.

Reply:
The preprocessor directives were switched with static constexpr uint8_t